pkgs/servers/web-apps/netbox/generic.nix

```diff
@@ -57,7 +57,7 @@
         markdown-include
         netaddr
         pillow
-        psycopg2
+        psycopg
```

Comment:
Won't this break netbox_3_5, which people could still want to migrate from?

Reply:
Unsure, the netbox_3_5 tests don't evaluate as they are eol. I need to figure out how I can run them.

Reply:
@xanderio you can run them with `NIXPKGS_ALLOW_INSECURE=1 nix build --impure '.#nixosTests.netbox_3_5'` if you have flakes enabled.

Without flakes enabled, you can do `NIXPKGS_ALLOW_INSECURE=1 nix-build -A 'nixosTests.netbox_3_5'`

And the same for the migration test.